rules/do-you-have-the-best-github-issue-template/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+archivedreason:
+title: Do you have the best GitHub Issue template
+guid: ba8f3441-139a-4638-9fc8-144408902104
+uri: do-you-have-the-best-github-issue-template
+created: 2021-07-06T12:44:34.0000000Z
+authors:
+  - title: Tom Bui
+    url: https://www.ssw.com.au/people/tom-bui
+related:
+- do-you-create-a-sprint-forecast-aka-the-functionality-that-will-be-developed-during-the-sprint
+
+---
+### Do you have the best GitHub Issue template
+
+GitHub issues (classic) offers a great way of raising issues within the projects. However, it can be difficult to distinguish whether the issue is about a bug fix, feature request or just a topic question.
+Therefore, GitHub issue templates should be used to help standardize and categorize different issue queries
+
+<!--endintro-->
+
+Based on [Configuring issue templates for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
```

Comment:
1. Can you add a link to an SSW repo with our configured templates (if one doesn't exist - create it)

Idea behind this is that other developers can grab the templates and our projects are consistent.

Reply:
> Can you add a link to an SSW repo with our configured templates

Done - added a link to [SSW CodeAuditor issues(https://github.com/SSWConsulting/SSW.CodeAuditor/issues/new/choose) as standard issue template

Reply:
sorry if it wasn't clear enough - that link just goes to creating a new issue on CodeAuditor

I think there should be a repo like `ssw.github.template` (marked as a template) that people can use as a base to build their project off - in this instance they would copy the templates out of the .github/ISSUE_TEMPLATE directory.

If this isn't clear please call.

Reply:
Done - added link to ssw github issue template